### ISSUE TITLE
Implementation with Firebase Admin

### ISSUE THREAD

**Issue by Sn0wfreezeDev** [jsayol/FireSQL#5]
Hi,

thanks again for this handy framework. 
I recently tried implementing it in a node environment. When I try to initialise it with a Firestore object which I have retrieved with the firebase admin framework it fails. Both frameworks should generally have the same API, but the types are different. 
So I think the issue is the type comparison here in firesql.js 

```
 if (typeof refOrOptions === 'object') {
            if (refOrOptions instanceof app_1.default.firestore.DocumentReference) {
                this._ref = refOrOptions;
            }
            else if (refOrOptions instanceof app_1.default.firestore.Firestore) {
                this._ref = refOrOptions.doc('/');
            }
            else if (!options) {
                this._options = refOrOptions;
            }
            else {
                throw new Error('With options as the second parameter, the first parameter ' +
                    'needs to be a path string or a Firestore reference.');
            }
```

Thank you in advance, 
Alex

**Comment by jsayol** [comment]
Hi Alex, thanks for pointing this out! You're right, at the moment only the client SDK is supported. I think using the admin SDK is a perfectly good use case though.

This should be an easy change, I'll let you know when a new version is out supporting this :-)

**Comment by jsayol** [comment]
Hi everyone! Sorry for the delay about this. I just published a new version (1.1.0) that adds support for the Admin SDK.

With it you can use the Admin SDK with `firesql.query()` without any problem.

If you're using `firesql.rxQuery()` you might run into some issues due to a current limitation of RxFire when using it with the Admin SDK. I already opened a pull request to fix this but I don't know when (or if) it will make it into the package. You can track the PR here: https://github.com/firebase/firebase-js-sdk/pull/1758

I hope this helps!
